twoliter/src/cmd/make.rs

```diff
@@ -1,11 +1,21 @@
 use crate::cargo_make::CargoMake;
-use crate::lock::Lock;
+use crate::lock::{Lock, LockedSDK};
 use crate::project::{self};
 use crate::tools::install_tools;
 use anyhow::Result;
 use clap::Parser;
 use std::path::PathBuf;
 
+// Some subcommands do not require kits and thus do not need to resolve and verify them against the
+// lockfile.
+//
+// Avoiding that resolution can be useful in CI situations where images are already built and we
+// need to perform additional operations using the SDK.
+//
+// `twoliter make` targets in the following list will skip kit validation IFF they also explicitly
+// list their SDK dependency in Twoliter.toml.
+const SKIP_KIT_VALIDATION_TARGETS: &[&str] = &["repack-variant", "repo", "fetch-sdk"];
```

Comment:
Seems like we would want to list all of the repo, AMI, ssm, and OVA operations in this list:
* repo
* validate-repo
* check-repo-expirations
* refresh-repo
* ami
* ami-public
* ami-private
* grant-ami
* revoke-ami
* validate-ami
* ssm
* promote-ssm
* validate-ssm

Optionally:
* upload-ova
* vmware-template

Unsure:
* all the test related tasks
* all the clean related tasks

I know allowlists are preferred over denylists, but we only really need kit validation to occur in the places where we're building something - `build-package`, `build-variant`, and `build-all`. So it might be better to just enforce it there?